### ISSUE TITLE
Requesting more information about PSiTE

### ISSUE THREAD

**Issue by sagnikbanerjee15** [hchyang/CSiTE#3]
Hello,

I was wondering if PSiTE can be used to generate data for 10X single-cell?

Thank you.

**Comment by hchyang** [comment]
No, it can not. PSiTE employs ART/WesSim to simulate WGS/WES NGS reads. As these tools can not simulate 10X reads, PSiTE can not generate 10X data.
